common/tags/src/test/java/org/apache/spark/tags/GitHubActionsUnstableTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.tags;
+
+import org.scalatest.TagAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@TagAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface GitHubActionsUnstableTest { }
```

Comment:
I temporarily added this tag to exclude some tests being failed in GitHub Actions. We should ideally remove this tag and reenable the tests skipped.

FWIW, the failed tests are:
- `ThriftServerQueryTestSuite`
-  `ThriftServerWithSparkContextSuite`
- `VersionSuite`

these do not happen in the master branch, and Jenkins still tests them so would not be a big deal.

`ThriftServerQueryTestSuite` and `ThriftServerWithSparkContextSuite` were fixed, IIRC, with an invasive fix that only landed to the master.

`VersionSuite` failure, I remember this but I could not figure out the root cause at that time.

Reply:
cc @dongjoon-hyun FYI